sourceCode.py

```diff
@@ -46,9 +39,9 @@
     "me"  : "you"
     }
   
```

Comment:
Maybe write a comment about what this dictionary is doing. Does it recognize one of the keys as input and produce the same sentence but with the value to describe what the user said?